### PR TITLE
Branding and mobile

### PR DESCRIPTION
**Pull request by pankajpatel**

### CODE REVIEW

**Comment by pankajpatel** [comment]
WIP

File: components/sandbox-manager/sandbox-manager.html.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
-module.exports = () => `
+module.exports = scope => `
   <style>
+<<<<<<< Updated upstream
```
Comment (nogizhopaboroda) [inline components/sandbox-manager/sandbox-manager.html.js]: unresolved conflict

File: components/sandbox-manager/sandbox-manager.js [hunk 2]
```diff
@@ -11,49 +11,111 @@ const template = require('./sandbox-manager.html');
...
+const SIZES = {
+  width: 80,
+  height: 75,
+  gutter: 8,
+  breakpoint: 600,
+};
 
 const createElement = (name, attributes) => {
   const $el = document.createElement(name);
+  console.log($el.type, $el)
```
Comment (nogizhopaboroda) [inline components/sandbox-manager/sandbox-manager.js]: forgotten consolelog

File: components/sandbox-manager/sandbox-manager.js [hunk 3]
```diff
@@ -11,49 +11,111 @@ const template = require('./sandbox-manager.html');
...
 class ManagerApp extends HTMLElement {
   connectedCallback() {
-    this.innerHTML = template();
-
-    const stories = getStories();
-    const firstStory = stories[Object.keys(stories)[0]];
+    this.mobile = this.hasAttribute('mobile');
+    this.branding = JSON.parse(this.getAttribute('branding') || '{}');
+    this.removeAttribute('branding');
+    this._stories = getStories();
```
Comment (nogizhopaboroda) [inline components/sandbox-manager/sandbox-manager.js]: this variable is not used anywhere

File: templates/manager_head.html.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-module.exports = () => `
+const config = require('../config');
```
Comment (nogizhopaboroda) [inline templates/manager_head.html.js]: please no dependencies other than template parts here

File: templates/preview_page.html.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
 const commonTemplate = require('./common.html');
 const previewTemplate = require('./preview.html');
+const config = require('../config');
```
Comment (nogizhopaboroda) [inline templates/preview_page.html.js]: same as in comment above - no config dependencies here

File: config.js [hunk 6]
```diff
@@ -4,6 +4,14 @@ const TARGET_DIR = process.cwd();
...
...MAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANil13Dcndag2KHXgNih18TYpdEsAAAAANSh10jMzZgU2KHWSAAAAATYpdZYAAAABNil1lgAAAAA2KHWSOCp0NzYodelAK4AMNih14zYodd42KHVZKyuABgAAgAIAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAEAAAAAAAAAATYodZI2KHWSAAAAAEArgAw2J3SiNih1/zUodc03J3SVAACAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEBAgAQ2KHXpNSd0XDUodYE2KHXdNih25jUodYs3K3MqAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA1KXRENih1/zYodfQ3KXWjNyh1RlUAVQMAAAAAAAAAADYodZIAAIACAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANSt1GDUpdVdAK4AMAAAAATYodZIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANyd0lQAAgAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABNil1lgAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAA3J3SVAACAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAE2KXWWAAAAAAAAAAAAAAAAAACAAgAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAEAAAAAAACAAgAAAAAAAAAAAAAAADYodcszM2YFNih1jgAAAAE2KHWSAAAAATYodZIAAAAANSd2jwAAAAA2KHWSAACAAjcpdckAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA//8AAP/EAAD/wAAA/8EAAAEAAABVEAAAPwAAAP8DAAA/BwAA//8AAD/nAAB/9wAAP+cAAFVXAAABRwAA//8AAA==',
```
Comment (nogizhopaboroda) [inline config.js]: this should be at least a file on file system. too crazy for default config

**Review by nogizhopaboroda — CHANGES_REQUESTED** [review]
Isn't it better to make branding matter of styles? We already have setup js file (`additional.js` by default). Styles file can be included there and icons and all the branding stuff can be changed there. Structure of branding config is too deep in this case, which makes `package.json` not the best place for it

**Pull request closed without merging**